tests/test_metrics_sdk_api.py

```diff
@@ -359,9 +358,7 @@ def test_raw_mql_string(self, test_dataset: str, tag_column: str) -> None:
 
         assert response.status_code == 200, data
         rows = data["data"]
-        assert len(rows) >= 180, rows
-
-        assert math.isnan(rows[0]["aggregate_value"])  # division by zero
+        assert len(rows) == 0
```

Comment:
this test used to give a bunch of nan rows. With the optimizer it returns no rows.

My thought was the division with conditional aggregates that have no matching columns must cause NAN. With the optimizer all rows would be filtered out making this expected behavior.

Reply:
i didnt investigate this too much but I thought since the results was all NaN before, having an empty result seems fine